src/thor/bidirectional_astar.cc

```diff
@@ -677,6 +663,7 @@ BidirectionalAStar::GetBestPath(valhalla::Location& origin,
     // the origin and destination and provides valid conditions for the reach-based pruning.
     bool force_forward = false;
     bool force_reverse = false;
+    // TODO: this is forcing if _any_ level is different, it doesn't distinguish levels
```

Comment:
meeh, will remove, not sure anymore if this a todo or not and I'm a bit sick of thinking about algos for now haha